src/test/capabilities_test.ts

```diff
@@ -17,8 +17,7 @@ import * as capabilities from '../capabilities';
 
 function assertBrowserCapabilities(
     userAgent: string, expect: capabilities.BrowserCapability[]) {
-  assert.deepEqual(
-      capabilities.browserCapabilities(userAgent), new Set(expect));
+  assert.sameMembers([...capabilities.browserCapabilities(userAgent)], expect);
```

Comment:
assert.sameMembers has sometimes very unhelpful output when the arrays are different lengths.  While cumbersome if written repeatedly, I would suggest the following since you're already writing this assert inside a function:
```
assert.deepEqual(
    [...capabilities.browserCapabilities(userAgent)].sort(), expect.sort());
```